Microwave.Test.Unit/UserInterfaceTest.cs

```diff
@@ -41,6 +41,24 @@ public void Setup()
                 light,
                 cooker);
         }
+        [Test]
+        public void Cooking_TimerButtonPressed_AddTimeCalled()
+        {
+            // set state to SETPOWER
+            powerButton.Pressed += Raise.EventWith(this, EventArgs.Empty);
+
+            // set state to SETTIME
+            timeButton.Pressed += Raise.EventWith(this, EventArgs.Empty);
+
+            // set state to COOKING
+            startCancelButton.Pressed += Raise.EventWith(this, EventArgs.Empty);
+
+            // Add 30 seconds to timer by pressing the timer button during cooking
+            timeButton.Pressed += Raise.EventWith(this, EventArgs.Empty);
```

Comment:
Lav det til et for-loop. Så kan man verificere, at det fungere korrekt hvis man klikker mere end 1 gang

Reply:
Forloop tilføjet, så det er nu verficeret, at knappen kan trykkes flere gange og cooker bliver kaldt det antal gange, som er blevet trykket.
Hvis knappen ikke trykkes, så sker der heller intet -> testet og OK.